src/common/keyring.cc

```diff
@@ -0,0 +1,117 @@
+
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+/*
+ * Ceph - scalable distributed file system
+ *
+ * Copyright (C) 2025 Clyso GmbH
+ *
+ * This is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License version 2.1, as published by the Free Software
+ * Foundation.  See file COPYING.
+ *
+ */
+
+#include "keyring.h"
+
+#include <keyutils.h>
+
+namespace ceph {
+
+#ifdef _WIN32
```

Comment:
i imagine compilation under windows won't be able to include keyutils.h. may not be worth the #ifdefs at all